File: dd-java-agent/instrumentation/junit-5.3/src/test/java/org/example/TestSucceedSkipEfd.java
package org.example;

import static org.junit.jupiter.api.Assertions.assertTrue;

import datadog.trace.api.civisibility.CIConstants;
import org.junit.jupiter.api.Tag;
import org.junit.jupiter.api.Tags;
import org.junit.jupiter.api.Test;

public class TestSucceedSkipEfd {

  @Test
  @Tags({@Tag(CIConstants.Tags.SKIP_EFD_TAG)})
  public void test_succeed() {
    assertTrue(true);
  }
}

File: dd-java-agent/instrumentation/scalatest/src/test/scala/org/example/TestSucceedSkipEfd.scala
package org.example

import datadog.trace.api.civisibility.CIConstants.Tags
import datadog.trace.api.civisibility.{CIConstants, InstrumentationBridge}
import org.scalatest.Tag
import org.scalatest.flatspec.AnyFlatSpec

object SkipEfdTag extends Tag(Tags.EFD_DISABLE_TAG)

class TestSucceedSkipEfd extends AnyFlatSpec {
  "test" should "assert something" taggedAs SkipEfdTag in {
    assert(2 + 2 > 3)
  }
}
